### PR TITLE
synchronize 4.5 version of json utils file back to 4.4

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
This mostly touches comments but one function is added

### CODE REVIEW

File: CRM/Utils/JSON.php [hunk 1]
```diff
@@ -39,16 +39,28 @@
 class CRM_Utils_JSON {
 
   /**
-   * Function to create JSON object
+   * Output json to the client.
+   * @param mixed $input
+   */
+  public static function output($input) {
+    header('Content-Type: application/json');
+    CRM_Utils_JSON::output($input);
```
Comment (colemanw) [inline CRM/Utils/JSON.php]: @eileenmcnaughton Whoops this is an infinite loop.
Funny because I made the same mistake when I first wrote this fn too :P

Comment (eileenmcnaughton) [inline CRM/Utils/JSON.php]: dang. I just copied the function from master - maybe I got your first version of it...

Comment (colemanw) [inline CRM/Utils/JSON.php]: Hmm, that's not the code I see in master...

Comment (eileenmcnaughton) [inline CRM/Utils/JSON.php]: found it - I ported this

https://github.com/civicrm/civicrm-core/commit/ecdef330eb57462e49d99c3b0c4e1c976c0e494a
but not the fix for it :-)

**Pull request merged**